src/coreclr/System.Private.CoreLib/src/System/Exception.CoreCLR.cs

```diff
@@ -163,14 +163,14 @@ internal void RestoreDispatchState(in DispatchState dispatchState)
         private string? _source;         // Mainly used by VB.
         private UIntPtr _ipForWatsonBuckets; // Used to persist the IP for Watson Bucketing
         private readonly IntPtr _xptrs;             // Internal EE stuff
-        private readonly int _xcode = _COMPlusExceptionCode;             // Internal EE stuff
+        private readonly int _xcode = CLRExceptionCode;             // Internal EE stuff
 #pragma warning restore CA1823, 414
 
         // @MANAGED: HResult is used from within the EE!  Rename with care - check VM directory
         private int _HResult;       // HResult
 
         // See src\inc\corexcep.h's EXCEPTION_COMPLUS definition:
```

Comment:
Is EXCEPTION_COMPLUS in scope of change?

Reply:
There are a bit more usages of COMPLUS in both comment and code, and I tend to include them in a new PR. What's your preference?

Reply:
sounds good, maybe a separate PR though.